src/components/Hero.js

```diff
@@ -25,25 +26,42 @@ function Hero({handleLogIn}) {
           <MarketingButton primary onClick={handleLogIn}>
             Login
           </MarketingButton>
-          <a href="">
-            <MarketingButton>
-              <Octicon verticalAlign="middle" icon={getIconByName("book")} />
-              {" "}Read The Story
-            </MarketingButton>
-          </a>
+          <MarketingButton primary onClick={handleLogIn}>
+            Signup
+          </MarketingButton>
         </Wrapper>
       </SubHero>
       <TrustSection>
-        {/*
         <small>Trusted By</small>
         <ul>
-          <li>ONE</li>
-          <li>TWO</li>
-          <li>THREE</li>
-          <li>FOUR</li>
+          <li>GraphQL</li>
+          <li>NextJS</li>
+          <li>Electron</li>
+          <li>Golang</li>
         </ul>
-        */}
       </TrustSection>
+      <QuoteSection>
+        <Link to="/">
+          <img style={{height: 50}} alt="open sauced" src={logo} />
+        </Link>
+        <p className="description">
+          Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo Yo
+        </p>
+        <div style={{marginTop: 50}}>
```

Comment:
replace the div with the `<Flex>` component here. It go responvise for small screens.

https://github.com/bdougie/open-sauced/blob/master/src/styles/Grid/Flex.js#L4-L9

Reply:
After adding Flex, you can remove the `left and `right` classNames

Reply:
Yay :tada:! It works, Thanks!